### PR TITLE
Fix flaky KeeperMap test

### PR DESCRIPTION
**Pull request by antonio2368**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
This is an automated comment for commit 57ada39ccf1615910afb48152a30ba7410152a36 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51764/57ada39ccf1615910afb48152a30ba7410152a36/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: tests/integration/helpers/network.py [hunk 1]
```diff
@@ -212,8 +219,8 @@ def _iptables_cmd_suffix(
 
     def __init__(
         self,
-        container_expire_timeout=50,
-        container_exit_timeout=60,
+        container_expire_timeout=120,
```
Comment (antonio2368) [inline tests/integration/helpers/network.py:222]: https://s3.amazonaws.com/clickhouse-test-reports/0/0f74d6d14dd01129979dc2361e080da6b150857c/integration_tests__release__[1_4].html

Container was erased in the middle of test

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
Sorry, I accidentially lost this commit while merging: https://github.com/ClickHouse/ClickHouse/pull/51764/commits/fb2affcae31afa1558706592860cc8f32e44ecde

**Comment by antonio2368** [comment]
https://github.com/ClickHouse/ClickHouse/pull/51764/commits/ff79ceaafe1d8cdcb1e5f46c41c126c4ada997eb is the important one and it seems it's on master

**Pull request merged**